google/data_source_google_billing_account.go

```diff
@@ -0,0 +1,111 @@
+package google
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+
+	"google.golang.org/api/cloudbilling/v1"
+	"google.golang.org/api/googleapi"
+)
+
+func dataSourceGoogleBillingAccount() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceBillingAccountRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"display_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"open": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+			"project_ids": {
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+		},
+	}
+}
+
+func dataSourceBillingAccountRead(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	name, nameOk := d.GetOk("name")
+	displayName, displayNameOk := d.GetOk("display_name")
+	if nameOk == displayNameOk {
```

Comment:
You can remove this check and use ConflictsWith in the schema definition above.

```go
"name": {
  Type: schema.TypeString
  ConflictsWith: []string{"display_name"},
  ...
}
```

Reply:
Done.